browser-interface/scripts/cdn-uploader-config.yml

```
@@ -0,0 +1,16 @@
+matches:
+  # Upload only the compressed variants of unity artifacts files
+  - match: "**/unity.framework.js"
+    variants:
+      - brotli
+  - match: "**/unity.data"
+    variants:
+      - brotli
```

This will make http://localhost not work, since it loads the uncompressed variant (brotli is not available outside https)
```suggestion
      - brotli
      - uncompressed
```

---

It also applies to the rest of the assets